tests/javascript/index.php

```diff
@@ -439,10 +439,14 @@ function PiwikTest() {
         var src = '<?php
             $src = file_get_contents('../../js/piwik.js');
             $src = strtr($src, array('\\'=>'\\\\',"'"=>"\\'",'"'=>'\\"',"\r"=>'\\r',"\n"=>'\\n','</'=>'<\/'));
+            $src = substr($src, strpos($src, '/* startjslint */'));
```

Comment:
This is a workaround to not run JSON3 through JSLINT as we cannot make it JSLINT compatible